intermediate_source/fx_conv_bn_fuser.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+"""
+(beta) Building a Conv/BN fuser in FX
+*******************************************************
+**Author**: `Horace He <https://github.com/chillee>`_
+
+In this tutorial, we are going to use FX to do the following:
+
+1) Find patterns of conv/batch norm in the data dependencies.
+2) Fold the batch norm statistics into the convolution weights.
```

Comment:
I think it's good to explicitly connect 1) and 2) to make it obvious, e.g. something like
2) For the patterns found in 1), fold the batch norm...